src/main/java/world/bentobox/bentobox/managers/BlueprintClipboardManager.java

```diff
@@ -131,8 +129,7 @@ public boolean load(User user, String fileName) {
      */
     public boolean save(User user, String newName) {
         clipboard.getBp().setName(newName);
-        File file = new File(blueprintFolder, newName);
-        if (saveBlueprint(file, clipboard.getBp())) {
+        if (saveBlueprint(clipboard.getBp())) {
```

Comment:
Clipboard#getBp() should be renamed to Clipboard#getBlueprint()

Reply:
Okay. I've also changes some other methods here to make it say blueprint.